README.md

```diff
@@ -1,23 +1,151 @@
 # kuadrantctl
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 
-Kuadrant configuration command line utility
+`kuadrantctl` is a CLI tool for managing Kuadrant configurations and resources.
 
 ## Installing
-Use `go install` to install the latest version of the library. This command will install the `kuadrantctl` binary executable in `$GOBIN` (defaulting to `$GOPATH/bin`).
 
-```
+`kuadrantctl` can be installed either by downloading pre-compiled binaries or by compiling from source. For most users, downloading the binary is the easiest and recommended method.
+
+### Installing Pre-compiled Binaries
+
+1. Download the latest binary for your platform from the [`kuadrantctl` Releases](https://github.com/Kuadrant/kuadrantctl/releases) page.
```

Comment:
Putting the kuadrantctl in quotes causes the link to split into 2 separate links both work just might cause weird things on website

Reply:
![Screenshot 2024-03-01 at 11 46 29](https://github.com/Kuadrant/kuadrantctl/assets/4467/03c030a4-af35-4a81-bef8-9f341ae92c77)

Seems to render OK, although I'll go through the rest for rendering sanity now that you mention it.

Reply:
oh deadly